cmd/syncthing/main.go

```diff
@@ -677,16 +677,12 @@ func syncthingMain() {
 			cfg.SetOptions(opts)
 			cfg.Save()
 		}
-		go usageReportingLoop(m)
-		go func() {
-			time.Sleep(10 * time.Minute)
-			err := sendUsageReport(m)
-			if err != nil {
-				l.Infoln("Usage report:", err)
-			}
-		}()
 	}
 
+	urMgr := newUsageReportingMgr(m)
```

Comment:
I guess we are too late with not using abbreviations, as we have a lot of *Svc structs already.